### PR TITLE
Feature/api

### PR DESCRIPTION
**Pull request by warriorzz**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -63,6 +63,9 @@ dependencies {
     implementation("io.ktor", "ktor-client-cio", ktorVersion)
     implementation("io.ktor", "ktor-client-json", ktorVersion)
     implementation("io.ktor", "ktor-serialization", ktorVersion)
+    implementation("io.ktor", "ktor-server-netty", ktorVersion)
```
Comment (NyCodeGHG) [inline build.gradle.kts:66]: Don't use netty when we are already using CIO

File: build.gradle.kts [hunk 2]
```diff
@@ -63,6 +63,9 @@ dependencies {
...
+    implementation("io.ktor", "ktor-server-core", ktorVersion)
```
Comment (NyCodeGHG) [inline build.gradle.kts:67]: Those are also unnecessary

File: src/main/kotlin/de/nycode/bankobot/twitch/WebhookRouter.kt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import io.ktor.serialization.*
+import io.ktor.server.cio.*
+import io.ktor.server.engine.*
+import io.ktor.util.*
+import mu.KotlinLogging
+
+internal val webhookLogger by lazy { KotlinLogging.logger("Webhooks") }
+
+/**
+ * Launches a ktor embedded server uses for receiving webhook notifications from the twitch api
```
Comment (NyCodeGHG) [inline src/main/kotlin/de/nycode/bankobot/twitch/WebhookRouter.kt:47]: Update those docs

**Review by NyCodeGHG — CHANGES_REQUESTED** [review]

**Review by DRSchlaubi — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (warriorzz) [reply build.gradle.kts:66]: Alright I forgot about that

**Review by NyCodeGHG — APPROVED** [review]

**Pull request merged**